client/web/src/cody/onboarding/CodyOnboarding.tsx

```diff
@@ -147,16 +157,16 @@ function PurposeStep({ onNext }: { onNext: () => void }): JSX.Element {
                     tabIndex={0}
                     className="flex-1 d-flex flex-column justify-content-center cursor-pointer align-items-center py-3 px-2"
                     onKeyDown={() => {
-                        eventLogger.log(EventName.CODY_ONBOARDING_PURPOSE_PERSONAL)
+                        eventLogger.log(EventName.CODY_ONBOARDING_PURPOSE_PERSONAL, { useCase: 'personal' })
                         onNext()
                     }}
                     onClick={() => {
-                        eventLogger.log(EventName.CODY_ONBOARDING_PURPOSE_PERSONAL)
+                        eventLogger.log(EventName.CODY_ONBOARDING_PURPOSE_PERSONAL, { useCase: 'personal' })
                         onNext()
                     }}
                 >
                     <PersonalIcon />
-                    <H3 className="mb-0 mt-2">Personal</H3>
+                    <H3 className="mb-0 mt-2">Personal Projects</H3>
```

Comment:
Nit, I think this is not a title
```suggestion
                    <H3 className="mb-0 mt-2">Personal projects</H3>
```